src/main/java/pingmebot/UI.java

```diff
@@ -0,0 +1,173 @@
+package pingmebot;
+
+import pingmebot.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with the user interactivity of the application.
+ */
+public class UI {
+    private final Scanner SC;
+
+    private ArrayList<String> response = new ArrayList<>();
+
+    /**
+     * Creates a UI object with the Scanner object intialised to read user's inputs.
+     */
+    public UI() {
+        this.SC = new Scanner(System.in);
+    }
+
+    /**
+     * Sends welcoming message to the user.
+     */
+    public void showWelcome() {
+        String greetingMsg = "Hello! I'm PingMeBot\n" + "What can I do for you?";
```

Comment:
Could extract this to be a constant, perhaps in a separate class or at the top of this class, so that it is easier to find and change if you need to.